.changes/fix-example-wasm.md

```diff
@@ -0,0 +1,5 @@
+---
+"wry": patch
+---
+
+Fix `custom_protocol` example caused an error when instantiating the Wasm module. `WebAssembly.instantiateStreaming` is not supported by WKWebView yet. `WebAssembly.instantiate` should be used instead.
```

Comment:
no need for a change file since this is internal change and doesn't concern the library users.